worker-sys/src/types/durable_object/sql_storage.rs

```diff
@@ -0,0 +1,24 @@
+use js_sys::JsString;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+extern "C" {
```

Comment:
There are few more methods for this class that we should implement: https://workers-types.pages.dev/#SqlStorage